erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -314,7 +317,7 @@ def update_value_in_dict(data, key, gle):
 				gle_map[gle.get(group_by)].entries.append(gle)
 			elif filters.get("group_by") == _('Group by Voucher (Consolidated)'):
 				key = (gle.get("voucher_type"), gle.get("voucher_no"),
-					gle.get("account"), gle.get("cost_center"))
+					gle.get("account"), gle.get("cost_center"), gle.get("against_voucher"))
```

Comment:
Please add docstrings and comments to this function. It will enhance the quality of our codebase